content/loader.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+chrome.runtime.sendMessage({ action: 'load' });
```

Comment:
Just wondering if we need it at all? Why can't we inject the scripts every time (keep it simple in `manifest.json`) and just avoid processing anything if it's not enabled?

Right now it seems that you're trying to re-implement "Enabled" checkbox on `chrome://extensions` and I'm not entirely sure what's the purpose of this (since we've agreed that the popup page will contain this option only temporarily).

I can imagine that in future it might be better to selectively enable/disable some features so we will rewrite it anyway.